### DISCUSSION TITLE
Can reports exclude issues where impact is minor?

### DISCUSSION THREAD

**Discussion by ghost** [TroyWalshProf/SeleniumAxeDotnet#232] [Q&A] [answered]
Hi

I'm running a report on web pages specifying the tags needed and then focusing on the 2 report types I am interested in:

var builder = new AxeBuilder(driver).WithTags("wcag2a", "wcag2aa", "wcag21a", "wcag21aa");
driver.CreateAxeHtmlReport(builder.Analyze(), filename, ReportTypes.Violations | ReportTypes.Incomplete);
AxeResult results = builder.Analyze();

This gives me a list of all the issues, but I'd also like to be able to ignore where impact = minor, so that the output report only contains critical and serious issues. Is this possible? I've read through the Selenium.Axe for .NET page but couldnt find anything.

Thanks, Steve

**Comment by TroyWalshProf** [comment, marked as answer]
This is a feature that should be added to [Axe.Core](https://github.com/dequelabs/axe-core) first.
In the short term you could get the results as a JSON file, remove unwanted impacts, and then load that JSON result.

**Reply by ghost** [reply]
Thanks for taking the time to reply, I will take a look at the JSON option in the short term.
